Any plans to support vitest 2+ ?

Any plans to support vitest 2+ ?

```
npm ERR! Could not resolve dependency:
npm ERR! peer vitest@">=0.25.0 <2.0.0" from testdouble-vitest@0.1.3
npm ERR! node_modules/testdouble-vitest
npm ERR!   testdouble-vitest@"*" from the root project
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
   "packageManager": "pnpm@8.14.1+sha256.2df78e65d433d7693b9d3fbdaf431b2d96bb4f96a2ffecd51a50efe16e50a6a8",
   "peerDependencies": {
     "testdouble": "^3.16.0",
-    "vitest": ">=0.25.0 <2.0.0"
+    "vitest": ">=0.25.0 <3.0.0"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^6.18.1",
@@ -50,10 +50,10 @@
     "eslint-plugin-promise": "^6.1.1",
     "prettier": "^3.2.2",
     "rimraf": "^5.0.5",
-    "testdouble": "^3.20.1",
+    "testdouble": "^3.20.2",
     "typescript": "^5.3.3",
-    "vite": "^5.0.11",
-    "vitest": "^1.2.0"
+    "vite": "^5.3.5",
+    "vitest": "^2.0.5"
   },
   "eslintConfig": {
     "extends": [

